pkgs/development/interpreters/erlang/R19.nix

```diff
@@ -45,6 +45,8 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     ./otp_build autoconf
+  '' + optionalString stdenv.isDarwin ''
+    substituteInPlace configure --replace "sw_vers" "/usr/bin/sw_vers"
```

Comment:
@grahamc I see it done here https://github.com/NixOS/nixpkgs/blob/5a31ad1bbf6acceba531f0995de4378370d87e31/pkgs/tools/admin/certbot/default.nix#L35 . Must be a Darwin-specific protocol for getting the system version.